package.json

```diff
@@ -18,12 +18,13 @@
     "plotly.js": "^1.58.4",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.3.1",
     "react-leaflet": "^3.1.0",
     "react-plotly.js": "^2.5.1",
     "react-query": "^3.6.0",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "4.0.1",
+    "react-scripts": "^5.0.0",
```

Comment:
Testing this out because it helps against npm audit things, havent seen a problem yet. See https://github.com/facebook/create-react-app/blob/main/CHANGELOG.md . Typescript errors shown in the browser look a bit nicer (the overlay with them is transparent), that's the only thing I noticed.